snuba/datasets/events.py

```diff
@@ -263,9 +263,9 @@ def __init__(self) -> None:
             write_schema=schema,
             stream_loader=KafkaStreamLoader(
                 processor=EventsProcessor(promoted_tag_columns),
-                default_topic="events",
-                replacement_topic="event-replacements",
-                commit_log_topic="snuba-commit-log",
+                default_topic=KafkaTopic("events"),
+                replacement_topic=KafkaTopic("event-replacements"),
+                commit_log_topic=KafkaTopic("snuba-commit-log"),
```

Comment:
It should be easier to add per-topic configuration values here now, like different partition counts or maximum message sizes, etc.

@BYK 👀